credentials/utopia-natcert-vcdm-v2/credential.json

```diff
@@ -0,0 +1,106 @@
+{
+  "@context": [
+    "https://www.w3.org/ns/credentials/v2",
+    "https://w3id.org/citizenship/v4rc1",
+    "https://w3id.org/security/data-integrity/v2",
+    {
+      "naturalizedOn": {
+        "@id": "https://w3id.org/citizenship#naturalizedOn",
+        "@type": "http://www.w3.org/2001/XMLSchema#dateTime"
+      }
+    },
+    {
+      "@protected": true,
+      "id": "@id",
+      "type": "@type",
+      "SvgRenderingTemplate2024": {
+        "@id": "https://w3id.org/vc/render-method#SvgRenderingTemplate2024",
+        "@context": {
+          "@protected": true,
+          "id": "@id",
+          "type": "@type",
+          "digestMultibase": {
+            "@id": "https://w3id.org/security#digestMultibase",
+            "@type": "https://w3id.org/security#multibase"
+          },
+          "name": "https://schema.org/name",
+          "mediaQuery": "https://w3id.org/vc/render-method#mediaQuery",
+          "template": "https://w3id.org/vc/render-method#template"
+        }
+      }
+    }
```

Comment:
@davidlehn, please help get these into some appropriate `rcNNN`  contexts instead of them being inline here.

Reply:
@dlongley I've swapped out `ceremonyDate` for `naturalizedOn` and am creating an example context for the render method stuff used in this credential. We can handle the more complicated upstreaming of this new Render Method later.